partner_compassion/models/partner_compassion.py

```diff
@@ -39,14 +39,6 @@ class ResPartner(models.Model):
     """
     _inherit = 'res.partner'
 
-    def _get_receipt_types(self):
```

Comment:
At line 36, try to extend GeoModel instead of model:
```
from odoo.addons.base_geoengine import geo_model
class ResPartner(geo_model.GeoModel):
```